main.tf

```diff
@@ -89,9 +89,14 @@ resource "google_cloud_run_v2_job" "job" {
   }
 }
 
+module "delivery-sa-name" {
+  source = "./short-unique-name"
+  name   = "${var.name}-dlv"
+}
+
 resource "google_service_account" "delivery" {
   project      = var.project_id
-  account_id   = "${var.name}-dlv"
```

Comment:
I'd probably just replace the `-dlv` suffix with a random string to fill what's left of the 30(?) characters the way we do in mono.

Reply:
OK, I swapped out hash for random string, and always use the suffix